Feat: add update exchange key page

SeijaHuang opened the pull request:
<!-- link to tickets -->
https://beequant.atlassian.net/jira/software/projects/CP/boards/1?selectedIssue=CP-11
---

## What happens here?
Added Update Exchange Key Page
-implemented update exchange key page
-added updated server action
-added Alert Message component
-added react-hot-toast for toast UI

screen shot:
1. light theme with Chinese:
<img width="1916" alt="image" src="https://github.com/user-attachments/assets/0bfaba2b-fca7-462b-90b4-5177114ba1c8">
2. dark theme with Eng 
<img width="1914" alt="image" src="https://github.com/user-attachments/assets/5300338f-2a0e-47ca-b69f-e4a46a1e4f95">
3. modal
<img width="1917" alt="image" src="https://github.com/user-attachments/assets/249ab5c3-3ded-4aca-8189-35b5651ed2e7">
4. update succeed
<img width="1915" alt="image" src="https://github.com/user-attachments/assets/e628368a-a6a1-4bae-967d-cfd4c2105e65">
5. invalid exchange key
<img width="1917" alt="image" src="https://github.com/user-attachments/assets/a454fbf5-e418-4f0e-b771-79d6391d0d45">


Resolve CP-11
## How do I know this is working?
1. first you need to add a exchange key to generate a key id
7. use the key id then route to localhost:3010/en/exchange/:id/update-exchange
8. then you can test
## Any notes?
1.  need first merge the exchange dashboard, otherwise it will not get any exchange key id and to let cancel to go back to the dashboard
2.  need to merge the delete exchange to make the delete button function as the name indicates
<!--
  New PR Checklist

- Have you tag the right reviewer?
- Have you dm the reviewer?
- Maybe @ the reviewer in channel just to be safe

- is CI passing in your pr?
- Have you update the ticket with the right status?
-->



Resolve CP-11
## How do I know this is working?
1. first you need to add a exchange key to generate a key id
2. use the key id then route to localhost:3010/en/exchange/:id/update-exchange
3. then you can test
## Any notes?
1.  need first merge the exchange dashboard, otherwise it will not get any exchange key id and to let cancel to go back to the dashboard
2.  need to merge the delete exchange to make the delete button function as the name indicates
<!--
  New PR Checklist

- Have you tag the right reviewer?
- Have you dm the reviewer?
- Maybe @ the reviewer in channel just to be safe

- is CI passing in your pr?
- Have you update the ticket with the right status?
-->

Chris-Wang reviewed (changes requested):
Need to resolve conflicts

SeijaHuang committed:
Added Update Exchange Key Page
-implemented update exchange key page
-added updated server action
-added Alert Message component
-added react-hot-toast for toast UI

Resolve CP-11

Chris-Wang reviewed (approved):
LGTM
